packages/middleware/src/integrations/registerIntegrations.ts

```diff
@@ -12,35 +11,55 @@ import type {
   IntegrationsLoaded,
 } from "../types";
 import { defaultErrorHandler } from "../errors/defaultErrorHandler";
+import { LoggerManager, wrapLogger } from "../loggerManager";
 
 export async function registerIntegrations(
   app: Express,
-  integrations: Integrations
+  integrations: Integrations,
+  loggerManager: LoggerManager
 ): Promise<IntegrationsLoaded> {
   return await Object.entries(integrations).reduce(
     async (prevAsync, [tag, integration]) => {
-      consola.info(`- Loading: ${tag} ${integration.location}`);
+      const rawLogger = loggerManager.get(tag);
```

Comment:
```suggestion
      const logger = loggerManager.get(tag);
```
meat can be raw, or data but not an object, what does it mean?